### PR TITLE
added onionURL icon Closes #199

### LINKED ISSUE #199: [Feature] Add icon with onion url hyperlink
Currently the url is displayed as the hyperlink over the name of the merchant. Thus there is only one url field possible to fill out, usually the clearnet site is most reasonable to input.

It would be nice to have an option to add the onion url as well. This could be displayed by a hyperlinked Tor onion icon next to the name of the merchant.

### PR DESCRIPTION
**Pull request by vaibhavgarg237**
Added onion URL icon in front of titles of organizations with beautiful bounce and flash animations. Fix #199 

Note: Tor Icon will only appear when onionUrl is available in List.js JSON. When adding new entries onionUrl should be provided.

### CODE REVIEW

**Review by vaibhavgarg237 — COMMENTED** [review]
-  ### Sample Image for Tor-Icon
image

-  ### Bounce animation + Flash animation on hover over Tor-Icon Video
https://user-images.githubusercontent.com/67408018/192119487-48b82b47-99c2-4e77-8114-c0b35310b0ac.mp4

**Comment by vaibhavgarg237** [comment]
> Hey, thanks for the contribution!
> 
> Would you mind removing the animation? We don't use these kinds of animations elsewhere and I think it becomes distracting fast, especially if there are multiple ones on the same page.

Okay, I will remove bounce animation EOD. I get that. Should we keep flash on hover? I think that will be intuitive!

**Comment by dennisreimann** [comment]
tbh I'd stick to what we have in the [design system](https://design.btcpayserver.org/) and also reuse the [onion svg from our icon sprite](https://github.com/btcpayserver/btcpayserver/blob/master/BTCPayServer/wwwroot/img/icon-sprite.svg?short_path=2d97a7f#L3) instead of introducing new things.

**Comment by vaibhavgarg237** [comment]
> tbh I'd stick to what we have in the [design system](https://design.btcpayserver.org/) and also reuse the [onion svg from our icon sprite](https://github.com/btcpayserver/btcpayserver/blob/master/BTCPayServer/wwwroot/img/icon-sprite.svg?short_path=2d97a7f#L3) instead of introducing new things.

Okay will do that

**Comment by vaibhavgarg237** [comment]
Updated...
https://onionicon.netlify.app/

###Dark Mode:
image
###Light Mode:
image

File: src/components/Directory/DirectoryItem.js [hunk 1]
```diff
@@ -12,6 +13,13 @@ function DirectoryItem(props) {
             ? icons[props.user.subType]
             : icons[props.user.type]}
           {props.user.name}
+          {props.user.onionUrl !== undefined && (
+            <a href={props.user.onionUrl} className="onionIcon" rel="noopener noreferrer">
+              <svg viewBox="0 0 1000 500">
+                <use href={iconSprite + "#onion"} />
+              </svg>
+            </a>
+          )}
```
Comment (dennisreimann) [inline src/components/Directory/DirectoryItem.js:22]: This link is nested within the other link, can you correct that?

Comment (vaibhavgarg237) [reply src/components/Directory/DirectoryItem.js:22]: okay, will check tmrw

Comment (vaibhavgarg237) [reply src/components/Directory/DirectoryItem.js]: > This link is nested within the other link, can you correct that?

Removed nested link

File: src/components/Directory/Directory.scss [hunk 2]
```diff
@@ -67,27 +67,37 @@
...
-
-          @media (max-width: $thirdBreak) {
-            font-size: 0.9rem;
-          }
-
-          &:hover {
-            color: $primaryGreen;
+          a {
+            // color: $mediumGrey;
+            color: $black;
```
Comment (dennisreimann) [inline src/components/Directory/Directory.scss:74]:
```suggestion
            color: inherit;
```

Comment (dennisreimann) [reply src/components/Directory/Directory.scss:74]: The link split change was good, but now the titles aren't readable in dark mode anymore.

grafik

Inheriting the color for the link fixes this.

**Review by dennisreimann — CHANGES_REQUESTED** [review]

**Review by dennisreimann — APPROVED** [review]
Thanks, looking good now!

**Comment by MaxHillebrand** [comment]
Sweet, thanks!

**Pull request merged**